--- C:\astor\examples\quixbugscompiled\levenshtein\src\java_programs\LEVENSHTEIN.java
+++ C:\astor\examples\quixbugscompiled\levenshtein\src\java_programs\LEVENSHTEIN.java
@@ -17,3 +17,3 @@
-			return 1 + java_programs.LEVENSHTEIN.levenshtein(source.substring(1), target.substring(1));
-		} else {
-			return 1 + java.lang.Math.min(java.lang.Math.min(
+			return 
+
+			java.lang.Math.min(
@@ -21 +21 @@
-			java_programs.LEVENSHTEIN.levenshtein(source.substring(1), target.substring(1))), 
+			java_programs.LEVENSHTEIN.levenshtein(source.substring(1), target.substring(1)));} else { 			return 1 + java.lang.Math.min(java.lang.Math.min(java_programs.LEVENSHTEIN.levenshtein(source, target.substring(1)), java_programs.LEVENSHTEIN.levenshtein(source.substring(1), target.substring(1))), 

